server/database.py

```diff
@@ -25,15 +26,19 @@ class User(AsceeModel, UserMixin):
     is_admin = props.Bool(indexed=True, default=False)
     is_recruiter = props.Bool(indexed=True, default=False)
     is_senior_recruiter = props.Bool(indexed=True, default=False)
+    is_submitted = props.Bool(indexed=True, default=False)
```

Comment:
Should we handle it like this, or using a status? Eh, whatever, this works.

Reply:
yes i thought that, but you can be submitted+claimed, submitted+escalated, submitted+unclaimed